analyzers/src/SonarAnalyzer.Common/Helpers/StringExtensions.cs

```diff
@@ -31,15 +31,17 @@ public static class StringExtensions
         /// Sequence of upper case letters is considered as single word.
         ///
         /// For example:
-        /// thisIsAName => this is a name
-        /// ThisIsSMTPName => this is smtp name
-        /// bin2hex => bin hex
-        /// HTML => html
-        /// SOME_value => some value
-        /// PEHeader => pe header
+        /// <list type="bullet">
```

Comment:
There's a duplication with `SonarAnalyzer.CFG\Helpers\StringExtensions.cs`

We should be able to change the namespace there to `SonarAnalyzer.Helpers`, move this change there and use only that one. In the past, CFG was not available in VB analyzer but it is now. So we can remove the duplication I think.